### ISSUE TITLE
Installing Reanimated v2 with RN 0.65.1 on iOS disables Hermes

### ISSUE THREAD

**Issue by brianathere** [software-mansion/react-native-reanimated#2330]
## Description
In a simple application using React Native 0.65.1, with Hermes enabled, on iOS, installing Reanimated v2 resulting in using the default JS runtime, not Hermes.

### Expected behavior

Hermes is found and enabled when using Reanimated v2

### Actual behavior & steps to reproduce

Repro:

npx react-native init AwesomeTSProject --template react-native-template-typescript
modify ios/Podfile, set :hermes_enabled => true
cd ios
pod install 
cd ..
npx react-native run-ios

Observe app starts and shows HEremes is enabled
yarn add react-native-reanimated@2
cs ios
pod install
cd ..
npx react-native run-ios

Observe now that Hermes is disabled

## Package versions

    "react-native": "0.65.1",
    "react-native-reanimated": "^2.2.0"

## Affected platforms

- [ ] Android
- [x] iOS
- [ ] Web

**Comment by brianathere** [comment]
Looking at UIResponder+Reanimated.mm I see a warning that looks related:
```
Category 'Reanimated' does not conform to protocol 'RCTBridgeDelegate'
```

**Comment by brianathere** [comment]
Seems this may be fixed by https://github.com/software-mansion/react-native-reanimated/commit/54e5e13b2bd818178fd0f924fa975f079c1e7f79, however other parts of our application aren't currently working with Reanimated 2.3 builds so I can't fully verify.

**Comment by duylvh** [comment]
I have the same issue with RN 0.65.1 and Reanimated v2.2.0

**Comment by taschik** [comment]
There is already an issue tracking this here: https://github.com/software-mansion/react-native-reanimated/issues/2313

**Comment by piaskowyk** [comment]
Hey @brianathere 👋  The version 2.2.0 doesn't support `react-native@0.65`. If you want to use Reanimated with RN65 you should update it to 2.2.1 or 2.3.0-alpha.2+

We have just released 2.2.1 🎉
